CI on main failing due to eslint error

```
$ eslint . --ignore-pattern '!.eslintrc.js' --ext js,ts,tsx,jsx --fix

Oops! Something went wrong! :(

ESLint: 8.48.0

ESLint couldn't find the config "plugin:@typescript-eslint/recommended-type-checked" to extend from. Please check that the name of the config is correct.

The config "plugin:@typescript-eslint/recommended-type-checked" was referenced from the config file in "/home/runner/work/rails-template/rails-template/tmp/builds/defaults-demo/node_modules/eslint-config-ackama/@typescript-eslint.js".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
./ci/bin/build-and-test:56:in `system': Command failed with exit 1: CONFIG_PATH="/home/runner/work/rails-template/rails-template/ackama_rails_template.config.yml" RACK_ENV=development RAILS_ENV=development rails new "defaults-demo" -d postgresql --skip-javascript -m "/home/runner/work/rails-template/rails-template/template.rb" (RuntimeError)
	from ./ci/bin/build-and-test:56:in `block in <main>'
	from ./ci/bin/build-and-test:48:in `chdir'
	from ./ci/bin/build-and-test:48:in `<main>'
Error: Process completed with exit code 1.
```

--- a/variants/frontend-base-typescript/template.rb
+++ b/variants/frontend-base-typescript/template.rb
@@ -23,8 +23,8 @@ types_packages = %w[
 
 yarn_add_dependencies types_packages + %w[@babel/preset-typescript typescript]
 yarn_add_dev_dependencies %w[
-  @typescript-eslint/parser@5
-  @typescript-eslint/eslint-plugin@5
+  @typescript-eslint/parser
+  @typescript-eslint/eslint-plugin
 ]
 
 run "yarn install"

